### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by parv0888**
## Purpose
Fixes issues #7 , #22, #61, #62

## Changes
Fixes issues #7 , #22, #61, #62

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG. Assumed not necessary since this is bug fix PR.

## CLA acceptance

By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [x] I accept the above linked CLA.

### CODE REVIEW

**Comment by parv0888** [comment]
image

**Comment by parv0888** [comment]
image

**Comment by parv0888** [comment]
image

File: backend/Application/Api/GraphQL/Accounts/AccountToken.cs
Comment (parv0888) [reply backend/Application/Api/GraphQL/Accounts/AccountToken.cs]: Yes. 
.Net Type system lacks an in-built type to represent the complete range. 
This has been converted to `System.Numerics.BigInteger`.

Comment (parv0888) [reply backend/Application/Api/GraphQL/Accounts/AccountToken.cs]: Kindly do have a look at the test cases
https://github.com/Concordium/concordium-scan/pull/67/files#diff-d2bc94ae3ce8b9ca27271b1856d6fd0c08383bf40205673e236fe0df7678aa40

File: backend/Application/Api/GraphQL/Extensions/ScalarTypes/BigIntegerScalarType.cs
Comment (abizjak) [inline backend/Application/Api/GraphQL/Extensions/ScalarTypes/BigIntegerScalarType.cs:5]: I am not sure I understand the purpose here. Can you please explain what this is doing for future maintainers.

File: backend/Application/Api/Rest/BalanceStatisticsController.cs
Comment (abizjak) [inline backend/Application/Api/Rest/BalanceStatisticsController.cs:49]: So the idea is to not show any decimals? Usually when we show amounts in CCD we show them as `123.456`.

Can we at least document why this choice here is being made?

**Review by abizjak — COMMENTED** [review]
Can you please update the CHANGELOGs and the version number of frontend and backend as well please?

File: backend/Application/Api/Rest/BalanceStatisticsController.cs
Comment (parv0888) [reply backend/Application/Api/Rest/BalanceStatisticsController.cs:49]: @abizjak Sure, 
@mh-concordium asked us to make this change, probably because this is needed for integration into other exchanges.

Comment (parv0888) [reply backend/Application/Api/Rest/BalanceStatisticsController.cs:49]: @mh-concordium : Kindly elaborate further

File: backend/Application/Api/GraphQL/Extensions/ScalarTypes/BigIntegerScalarType.cs
Comment (parv0888) [reply backend/Application/Api/GraphQL/Extensions/ScalarTypes/BigIntegerScalarType.cs:5]: Have added it to doc comment.

**Comment by parv0888** [comment]
> Can you please update the CHANGELOGs and the version number of frontend and backend as well please?

Yes Surely, 
Its done

File: backend/Application/Api/Rest/BalanceStatisticsController.cs
Comment (mh-concordium) [reply backend/Application/Api/Rest/BalanceStatisticsController.cs:49]: @parv0888 could you please link to what issue is this fixing?

Comment (mh-concordium) [reply backend/Application/Api/Rest/BalanceStatisticsController.cs:49]: I don't remember I made such a feedback at any point and issue 66 was created by you @parv0888 .  Looking at issues, I did create issue where I suggested showing all the decimals and aligning number to the right.

Comment (parv0888) [reply backend/Application/Api/Rest/BalanceStatisticsController.cs:49]: @mh-concordium 
This is the issue that was communicated to me by @vj, We had a call about this.
I Created this issue so that its easier to document this on the PR. 
Im specifically unaware of the real reason why this was needed. So wanting your help in communicating this to @abizjak 
@coolmagic : Can you help?

File: backend/Application/Api/GraphQL/Extensions/ScalarTypes/BigIntegerScalarType.cs
Comment (abizjak) [reply backend/Application/Api/GraphQL/Extensions/ScalarTypes/BigIntegerScalarType.cs:5]: Looks good.

**Comment by parv0888** [comment]
@abizjak  : code to fix #66 has been removed

**Comment by abizjak** [comment]
> @abizjak : code to fix #66 has been removed

Great, thanks.

**Review by abizjak — APPROVED** [review]

**Comment by abizjak** [comment]
@parv0888 have you udpated the version number here already?

Can you merge this after doing so?

**Pull request merged**
